benchmark/main.cpp

```diff
@@ -179,6 +179,8 @@
         mbedtls_printf( "FAILED: -0x%04x\r\n", -ret );
 #endif
 
+#if defined(CoreDebug_DEMCR_TRCENA_Msk) && defined(DWT_CTRL_CYCCNTENA_Msk)
+/* DWT cycle counter is available (Cortex-M3, Cortex-M4) */
```

Comment:
The code always prints kB/s. I only changed the additional printing of cycles/byte, which aren't meaningful across platforms. The only fully portable alternative is not to have a cycle count at all, AFAICT. So this is the best I know how to do at the moment, but I'm open to suggestions.